README.md

```diff
@@ -50,3 +50,45 @@ To run the tests using Poetry and Pytest, stop the app running above and run the
 ```bash
 $ poetry run pytest
 ```
+
+## Running the application using docker
+Use following commands to run docker image todo-app
+
+## Running the application in single-stage docker file
+```bash
+docker build -t todo-app .
+
+Pass environment variables as a file, publish the application on port 5000 and listen on port 5000
+docker run --env-file .env -p 5000:5000 
+```
+## Add gunicorn for running in Production environment
+```bash
+poetry add gunicorn
+```
+
+## Running the application in multi-stage docker file in production
+```bash
+docker build -t todo-app:prod --target production .
+
+Pass environment variables as a file, publish the application on port 5000 and listen on port 5000
+docker run --env-file .env -p 5000:5000 todo-app:prod
+```
+
+## Running the application in multi-stage docker file in development
+```bash
+docker build -t todo-app:dev --target development .
+
+Pass environment variables as a file, publish the application on port 5000 and listen on port 5000
+docker run --env-file .env -p 5000:5000 todo-app:dev
```

Comment:
Can you try adding the correct option to run the container with a bind mount? It would be much more convenient for developers - it means the app files are shared with the container as it runs, so you can update files and see the app update with a hot reload instead of needing to rebuild/rerun the image/container.

Or if you get the docker-compose file working, your README could just tell people to run `docker compose up todo_app_dev`